.changes/next-release/Feature-b17cee45-65a7-4dea-a358-6d4a4c9ecf67.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "Expose listConnections API in AWS Toolkit"
```

Comment:
Please mention it in the docs: https://github.com/aws/aws-toolkit-vscode/blob/master/docs/api.md